File: tests/resources/generator/dependent_eqns/code.py
from math import *


def initialize_constants(states, variables):
    states[0] = 0.0


def compute_computed_constants(variables):
    pass


def compute_rates(voi, states, rates, variables):
    rates[0] = 1.0


def compute_variables(voi, states, rates, variables):
    variables[0] = 2.0*voi
    variables[1] = 3.0*variables[0]

File: tests/generator/resources/ode_constant_on_rhs/code.c
#include <math.h>

void initializeModel(double *states, double *variables)
{
    states[0] = 1.0;
}

void computeConstantEquations(double *variables)
{
}

void computeRateEquations(double voi, double *states, double *rates, double *variables)
{
    rates[0] = 1.0;
}

void computeAlgebraicEquations(double voi, double *states, double *rates, double *variables)
{
}

void computeStateOrRateBasedAlgebraicEquations(double voi, double *states, double *rates, double *variables)
{
}
